satellite/rewards/rewards.go

```diff
@@ -5,9 +5,28 @@ package rewards
 
 import (
 	"context"
+	"fmt"
+	"math"
+	"strconv"
 	"time"
 )
 
+// ToCents converts USD credit amounts to cents.
+func ToCents(dollar int) int {
+	cents := float64(dollar) * 100
```

Comment:
Why do we need to convert `dollar` into `floast64` here and then convert it back to `int` in return?

Reply:
`Math.Round ` only takes a `float64`

Reply:
If we are using integer. Why do we need to round it?

Reply:
You're right, we don't. I removed it and pushed.